src/pages/SitePermissionPage.qml

```diff
@@ -0,0 +1,94 @@
+/****************************************************************************
+**
+** Copyright (c) 2020 Open Mobile Platform LLC.
+**
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.6
+import Sailfish.Silica 1.0
+import Sailfish.WebView.Controls 1.0
+
+Page {
+    id: page
+
+    property string title
+    property string url
```

Comment:
Could these be aliases, rather than properties? If so, the `id: page` can be dropped.

Reply:
it used in some  places